clang/test/Lexer/raw-string-dlim-newline.cpp

```diff
@@ -0,0 +1,5 @@
+// RUN: not  %clang_cc1 -E %s 2>&1 | grep 'error: invalid newline character in raw string delimiter; use PREFIX( )PREFIX to delimit raw string'
```

Comment:
Please use `-fsyntax-only -verify` instead of `grep` for checking diagnostics. There are some examples in sibling files, but the gist of it is that you'll need a `// expected-error@+1{{invalid newline character...}}` on the line before the declaration.

Reply:
mind also adding a test for the `err_invalid_char_raw_delim` case while you're here?